back/engines/commercial/report_builder/app/services/report_builder/queries/most_reacted_ideas.rb

```diff
@@ -0,0 +1,24 @@
+module ReportBuilder
+  class Queries::MostReactedIdeas < ReportBuilder::Queries::Base
+    protected
+
+    def run_query(project_id:, number_of_ideas:, phase_id: nil, **_other_props)
+      project = Project.find(project_id)
+      phase = phase_id ? Phase.find(phase_id) : project.phases.first
+      ideas = phase.ideas.order(likes_count: :asc).limit(number_of_ideas)
```

Comment:
@luucvanderzee Currently, we sort by `likes_count` in ascending order, which actually means least liked ideas
https://github.com/CitizenLabDotCo/citizenlab/blob/08a97ee4e9caf17f0c5249add45e07b15fd139c9/front/app/containers/Admin/reporting/hooks/useMostReactedIdeas.ts#L16
https://github.com/CitizenLabDotCo/citizenlab/blob/08a97ee4e9caf17f0c5249add45e07b15fd139c9/back/app/services/sort_by_params_service.rb#L28

Should it be `likes_count: :desc`?

Reply:
Ah, no that seems like a mistake in the hook. The backend code of the sort service looks correct to me: if I specify `likes_count` I would expect the one with the highest count to come first and with the lowest count to come last, i.e. descending. If I specify `-likes_count` the opposite, i.e. ascending.

So we should indeed do

```ruby
ideas = phase.ideas.order(likes_count: :desc).limit(number_of_ideas)
```